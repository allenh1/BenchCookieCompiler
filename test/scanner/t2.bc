[ int; 7; 7.43; ]
