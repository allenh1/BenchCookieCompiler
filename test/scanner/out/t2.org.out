{<OBRACKET>: [}
{<INT>: int}
{<SEMICOLON>: ;}
{<INTEGER>: 7}
{<SEMICOLON>: ;}
{<FLOAT>: 7.43}
{<SEMICOLON>: ;}
{<CBRACKET>: ]}
